django_comparison_dashboard/datamodel/datapackage.json

```diff
@@ -41,6 +41,8 @@
         {"name": "groups", "description": "Free classification with key-value pairs", "type": "array", "unit": null},
         {"name": "input_commodity", "description": "It describes any type of energy or energy carrier (e.g. electricity, heat, solar radiation, natural gas, ...) that enters a technology.", "type": "string", "unit": null},
         {"name": "output_commodity", "description": "It describes any type of energy or energy carrier (e.g. electricity, heat, hydrogen, LNG, CO2, ...) that exits a technology.", "type": "string", "unit": null},
+        {"name": "input_groups", "description": "Free classification with key-value pairs", "type": "array", "unit": null},
```

Comment:
Why does the description say "key-value" pairs, although we give the values in a list?

Reply:
Don't know where this comes from - I just copy-pasted it. will adapt